test/UnitTest/v2/test_casesNGSIv2.py

```diff
@@ -33,149 +34,153 @@ def test_getAllSubscription():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	response=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	assert response.status_code==201
+def test_Subscription():
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        response=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        assert response.status_code==201
 
 #testCase 9
-#update request wit create action 
+#update request wit create action
 
 '''
   Testing update request with update action
 '''
 def test_update_request_with_update_action():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	updateresponse=requests.post(url,data=json.dumps(v2data.updateDataWithupdateaction),headers=headers)	
-	assert updateresponse.status_code == 201
-  
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        updateresponse=requests.post(url,data=json.dumps(v2data.updateDataWithupdateaction),headers=headers)
+        assert updateresponse.status_code == 201
+
 #testCase 10
 '''
   Testing update request with Delete request
 '''
 
 def test_upadte_request_with_delete_action():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	updateresponse=requests.post(url,data=json.dumps(v2data.deleteDataWithupdateaction),headers=headers)
-	assert updateresponse.status_code==201
-	
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        updateresponse=requests.post(url,data=json.dumps(v2data.deleteDataWithupdateaction),headers=headers)
+        assert updateresponse.status_code==201
+
+
 #testCase 11
 
 '''
   Testing update request with create action
 '''
 
-def test_update_request_with_create_action():	
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	updateresponse=requests.post(url,data=json.dumps(v2data.createDataWithupdateaction),headers=headers)
-	assert updateresponse.status_code==201
+def test_update_request_with_create_action():
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        subresponse=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        updateresponse=requests.post(url,data=json.dumps(v2data.createDataWithupdateaction),headers=headers)
+        assert updateresponse.status_code==201
 
 #testCase 12
 '''
```

Comment:
Notification should be received by subscriber, test notification for both the cases:
a)  when subscription is created first and then data are posted later,
b)  when data already exist, and then subscription is created for existing data